### PR TITLE
jade-2023-raw-datasets: EOS file information for notes

### PR DESCRIPTION
**Pull request by tiborsimko**
Adds file information for JADE notes.

Closes cernopendata/opendata.cern.ch#3492.

### CODE REVIEW

**Comment by rdebrand** [comment]
Is there also an entry for CN 71? Looks like it's missing from the file information.

**Comment by tiborsimko** [comment]
> Is there also an entry for CN 71? Looks like it's missing from the file information.

There was no PDF file for 71 in the "Separated" directory.  Perhaps it still needs to be constructed from the "Scanned" files?  Perhaps @andriish knows?

**Comment by andriish** [comment]
Hi @rdebrand, @tiborsimko,

CN71 is there https://github.com/andriish/JADE/blob/master/doc/ComputingNotes/Separated/71.pdf,
but for some reason I see that it was added couple commits after the bulk of the notes.

Best regards,

Andrii

**Review by andriish — APPROVED** [review]
Hi @tiborsimko,
this LGTM!

We have the "missing" notes 71 and 103:

https://github.com/andriish/JADE/blob/master/doc/ComputingNotes/Separated/103.pdf
and
https://github.com/andriish/JADE/blob/master/doc/ComputingNotes/Separated/71.pdf

How can we upload them to EOS?


Andrii

**Comment by andriish** [comment]
Basically the string to add are:

path=/eos/opendata/jade/documentation/notes/71.pdf size=125655 checksum=79baf953
path=/eos/opendata/jade/documentation/notes/103.pdf size=829048 checksum=b525b56a

File: jade-2023-raw-datasets/inputs/eos-file-information-notes.txt [hunk 1]
```diff
@@ -0,0 +1,116 @@
...
+path=/eos/opendata/jade/documentation/notes/64.pdf size=88352 checksum=799d9aae
+path=/eos/opendata/jade/documentation/notes/65.pdf size=98988 checksum=a98d257c
+path=/eos/opendata/jade/documentation/notes/66.pdf size=801251 checksum=89a4c4b9
+path=/eos/opendata/jade/documentation/notes/66a.pdf size=77194 checksum=ced0e952
+path=/eos/opendata/jade/documentation/notes/67.pdf size=691593 checksum=cbfcf4b4
+path=/eos/opendata/jade/documentation/notes/68.pdf size=233990 checksum=05ac7b52
+path=/eos/opendata/jade/documentation/notes/69.pdf size=459468 checksum=7e13c542
+path=/eos/opendata/jade/documentation/notes/7.pdf size=342016 checksum=9602a417
+path=/eos/opendata/jade/documentation/notes/70.pdf size=231213 checksum=e330deb0
+path=/eos/opendata/jade/documentation/notes/72.pdf size=127216 checksum=5687eb44
```
Comment (andriish) [inline jade-2023-raw-datasets/inputs/eos-file-information-notes.txt:85]:
```suggestion
+path=/eos/opendata/jade/documentation/notes/71.pdf size=125655 checksum=79baf953
```

File: jade-2023-raw-datasets/inputs/eos-file-information-notes.txt [hunk 2]
```diff
@@ -0,0 +1,116 @@
...
+path=/eos/opendata/jade/documentation/notes/63a.pdf size=52775 checksum=653c030c
+path=/eos/opendata/jade/documentation/notes/64.pdf size=88352 checksum=799d9aae
+path=/eos/opendata/jade/documentation/notes/65.pdf size=98988 checksum=a98d257c
+path=/eos/opendata/jade/documentation/notes/66.pdf size=801251 checksum=89a4c4b9
+path=/eos/opendata/jade/documentation/notes/66a.pdf size=77194 checksum=ced0e952
+path=/eos/opendata/jade/documentation/notes/67.pdf size=691593 checksum=cbfcf4b4
+path=/eos/opendata/jade/documentation/notes/68.pdf size=233990 checksum=05ac7b52
+path=/eos/opendata/jade/documentation/notes/69.pdf size=459468 checksum=7e13c542
+path=/eos/opendata/jade/documentation/notes/7.pdf size=342016 checksum=9602a417
+path=/eos/opendata/jade/documentation/notes/70.pdf size=231213 checksum=e330deb0
```
Comment (andriish) [inline jade-2023-raw-datasets/inputs/eos-file-information-notes.txt:84]:
```suggestion
path=/eos/opendata/jade/documentation/notes/71.pdf size=125655 checksum=79baf953
```

File: jade-2023-raw-datasets/inputs/eos-file-information-notes.txt [hunk 3]
```diff
@@ -0,0 +1,116 @@
+path=/eos/opendata/jade/documentation/notes/0Index.pdf size=137529 checksum=46c1eb5f
+path=/eos/opendata/jade/documentation/notes/10.pdf size=306074 checksum=cbb6b90c
+path=/eos/opendata/jade/documentation/notes/100.pdf size=360837 checksum=55a69faa
+path=/eos/opendata/jade/documentation/notes/101.pdf size=385882 checksum=10986d99
+path=/eos/opendata/jade/documentation/notes/102.pdf size=1254353 checksum=e5c800b9
```
Comment (andriish) [inline jade-2023-raw-datasets/inputs/eos-file-information-notes.txt:5]:
```suggestion
path=/eos/opendata/jade/documentation/notes/102.pdf size=1254353 checksum=e5c800b9
path=/eos/opendata/jade/documentation/notes/103.pdf size=829048 checksum=b525b56a
```

**Comment by tiborsimko** [comment]
> How can we upload them to EOS?

Thanks, I have downloaded them and added them to EOSPUBLIC.

**Pull request merged**